examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -66,8 +67,11 @@ export class TaskList extends DataObject implements ITaskList {
      * collection pattern -- see the contact-collection example for more details on this pattern.
      */
     private readonly tasks = new Map<string, Task>();
+    private _savedData: SharedMap | undefined;
```

Comment:
Consider also adding private get functions that are non-null (e.g. similar to collaborative-textarea for `text`: https://github.com/microsoft/FluidFramework/blob/main/examples/apps/collaborative-textarea/src/fluid-object/index.ts#L17).  This way in places below where you depend on it being non-null you can avoid using `?` operators.